burger.css

```diff
@@ -0,0 +1,58 @@
+   
+body {
+  background: #E5E5E5;
+}
+
+   .header__nav {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/MyBali/pullRequest?prid=5781003)